pkg/api/handlers/organization.go

```diff
@@ -176,11 +175,21 @@ func (oh *OrganizationHandler) PostProject(c echo.Context) error {
 		)
 	}
 
-	project := organizationmodel.Project{
-		Name:        request.Name,
-		Description: request.Description,
+	for _, project := range organizationRecord.Projects {
+		if project.Name == request.Name {
+			oh.logger.Debug("Client error",
+				zap.String("cause", "Non-unique project name"),
+			)
+			return apierrors.CustomError(
+				c,
+				http.StatusBadRequest,
+				apierrors.BadRequestError,
+			)
+		}
```

Comment:
create a `FindOne` method in the organization model that receives any query and do something like the following and check if it returns a record. If it returns a record we know that the name is already being used in a project for the organization

```Go
organizationModel.FindOne(context.Background(), bson.M{"$and": []bson.M{
			{"_id": organizationID},
			{"projects": bson.M{
			"$elementMatch": bson.D{{Key: "name", Value: request.Name}}
			}},
		}})
```